utils/linux-install-openresty.sh

```diff
@@ -80,4 +81,23 @@ else
     openresty="openresty-debug=$OPENRESTY_VERSION*"
 fi
 
-sudo apt-get install "$openresty" lua5.1 liblua5.1-0-dev openresty-openssl-debug-dev
+verlte() {
+    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
+}
+
+verlt() {
+    [ "$1" = "$2" ] && return 1 || verlte $1 $2
+}
+
+sudo apt-get install "$openresty" lua5.1 liblua5.1-0-dev
+
+if [ "$OPENRESTY_VERSION" == "default" ]; then
+    sudo apt-get install openresty-openssl111-debug-dev
+else
+    verlt $OPENRESTY_VERSION 1.17.8.1
```

Comment:
We can simply check if the version is `1.15.8.2` so there is no need to comment out `set -euo pipefail`

Reply:
Fixed.